New package: PowerModelsDistributionRDT v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: PowerModelsDistributionRDT
- Repository: https://github.com/lanl-ansi/PowerModelsDistributionRDT.jl
- Created by: @pseudocubic
- Version: v0.1.0
- Commit: 0af7a2b6d3f5f8e3e07f59520e1ba95c7276e8db
- Reviewed by: @pseudocubic
- Reference: https://github.com/lanl-ansi/PowerModelsDistributionRDT.jl/commit/0af7a2b6d3f5f8e3e07f59520e1ba95c7276e8db#commitcomment-147978356
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd636e3c347fcb476c1c903894fdc66913549501e0f597115d821f1a8756e52f3e91e25ab43b39666d88b4b7b0c7d4649181a474201d99a985e2191bff171a150ce59a32630e49010a895e10575812f25dd452c0b07886a0b56e88e2e2e9de6e09683350d078746ab09831c6b9f59dd487f1de80c101c65618534eeeb76304fc54652794bb9eca474259ff0caed97988a179a6091a6ec1e1d998f1c3ca4e559858af987571d6adb1ef5c0d8e87c5876f22686232ececbe7f0240038291311722529 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: PowerModelsDistributionRDT v0.1.0

UUID: 66cb097a-b461-11e9-2c74-17ced528deb4
Repo: https://github.com/lanl-ansi/PowerModelsDistributionRDT.jl.git
Tree: c03de68bdac9eeec5ce7324c4f1868ff862aa882

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
